sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -1099,6 +1099,30 @@ def __init__(self, **kwargs):
         self.has_header = kwargs.pop('has_header', False)
 
 
+class ArrowDialect(ArrowField):
+    """field of an arrow schema.
+
+    All required parameters must be populated in order to send to Azure.
```

Comment:
For the docstrings, I don't think we're sending to Azure, are we? I thought we are only using this format to serialize the output?

Reply:
Yes, we are sending this to Azure. This is to tell service the format we want the data to return in which format.